src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/VariantAnnotatorEngine.java

```diff
@@ -365,7 +366,12 @@ public VariantContext annotateContext(final VariantContext vc,
         final VariantContext annotated = builder.attributes(infoAnnotMap).make();
 
         // annotate db occurrences
-        return variantOverlapAnnotator.annotateOverlaps(features, variantOverlapAnnotator.annotateRsID(features, annotated));
+        try {
+            return variantOverlapAnnotator.annotateOverlaps(features, variantOverlapAnnotator.annotateRsID(features, annotated));
+        } catch (IllegalArgumentException e) {
```

Comment:
As we discussed in person, I think we should check that other errors that aren't intended to be caught are not caught here. I just want to be sure that we don't change the default behavior for unrelated variant annotations if they would have failed here.